packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -50,21 +68,64 @@ class _ErrorAwareText extends StatelessWidget {
   const _ErrorAwareText({
     required this.text,
     required this.isError,
+    required this.controller,
+    required this.link,
   });
   final String text;
   final bool isError;
+  final DeepLinksController controller;
+  final LinkData link;
 
   @override
   Widget build(BuildContext context) {
     return Row(
       children: [
         if (isError)
-          Padding(
-            padding: const EdgeInsets.only(right: denseSpacing),
-            child: Icon(
-              Icons.error,
-              color: Theme.of(context).colorScheme.error,
-              size: defaultIconSize,
+          DevToolsTooltip(
+            padding: const EdgeInsets.only(
+              top: defaultSpacing,
+              left: defaultSpacing,
+              right: defaultSpacing,
+            ),
+            preferBelow: true,
+            richMessage: WidgetSpan(
+              child: SizedBox(
+                width: 344,
```

Comment:
put this into a named const. This seems like an strange width... is there a reason we are using a fixed width instead of letting this be dynamically sized with the screen? For example, does this fixed width make the display strange when the font size is increased or the screen size changes?

Reply:
This width is for the tooltip. if not add a fixed width the default behavior is to show the string in one line and the tooltip will be super long and looks strange. 

Because the tooltip is floating, I think it's ok to not change its size when screen size or font size changes.

Reply:
Noted. You can add new line characters into the text to make it multi-line `/n`